### PR TITLE
Skip PITR test for PXC 5.7

### PR DESCRIPTION
**Pull request by tplavcic**

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| recreate | passed |
| haproxy | passed |
| storage | passed |
| upgrade-haproxy | passed |
| big-data | passed |
| scaling-proxysql | passed |
| init-deploy | passed |
| self-healing | passed |
| security-context | passed |
| restore-to-encrypted-cluster | passed |
| upgrade-proxysql | passed |
| limits | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| smart-update | passed |
| affinity | passed |
| operator-self-healing | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| pitr | passed |
| upgrade-consistency | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |
| scheduled-backup | failed |

File: e2e-tests/pitr/run [hunk 1]
```diff
@@ -6,6 +6,15 @@ set -o xtrace
 test_dir=$(realpath $(dirname $0))
 . ${test_dir}/../functions
 
+if [[ "${IMAGE_PXC}" == *"percona-xtradb-cluster-operator"* ]]; then
+    PXC_VER=$(echo -n "${IMAGE_PXC}" | $sed -r 's/.*([0-9].[0-9])$/\1/')
+else
+    PXC_VER=$(echo -n "${IMAGE_PXC}" | $sed -r 's/.*:([0-9]+\.[0-9]+).*$/\1/')
+fi
+if [[ "${PXC_VER}" == "5.7" ]]; then
```
Comment (delgod) [inline e2e-tests/pitr/run:14]: we can keep it simple
```suggestion
if [[ "$IMAGE_PXC" =~ 5\.7 ]]; then
```

**Review by delgod — APPROVED** [review]
comment up to you, but we have the similar code in `compare_kubectl` function

File: e2e-tests/pitr/run (hunk 1 above)
Comment (tplavcic) [reply e2e-tests/pitr/run]: Agree. I wanted to escape the issue with some images like `percona/percona-xtradb-cluster:5.7.32-31.47` where the last two numbers change often, but not sure how likely is it to happen to have a match.

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | passed |
| big-data | passed |
| scaling | passed |
| upgrade-haproxy | passed |
| storage | passed |
| recreate | passed |
| init-deploy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| limits | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | passed |
| smart-update | passed |
| demand-backup | passed |
| affinity | passed |
| one-pod | passed |
| operator-self-healing | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| pitr | passed |
| users | passed |
| upgrade-consistency | passed |
| demand-backup-encrypted-with-tls | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |
| scheduled-backup | failed |

**Pull request merged**